schema/schemaspec/schemahcl/context_test.go

```diff
@@ -5,6 +5,7 @@ import (
 	"testing"
 
 	"ariga.io/atlas/schema/schemaspec"
+	"ariga.io/atlas/schema/schemaspec/internal/schemautil"
```

Comment:
A line break between internal imports and external.

```suggestion
	"ariga.io/atlas/schema/schemaspec/internal/schemautil"

```